controller/lib/hal/hal.h

```diff
@@ -61,11 +62,22 @@ limitations under the License.
 
 #else // !TEST_MODE
 
-#ifndef AVR
-#error "When running without TEST_MODE, expecting AVR to be defined"
+#if !defined(ARDUINO_AVR_UNO) && !defined(ARDUINO_NUCLEO_L452RE)
+#error "When running without TEST_MODE, expecting ARDUINO_AVR_UNO to be defined"
 #endif
 #include <Arduino.h>
 
+#ifdef ARDUINO_AVR_UNO
+#include <avr/wdt.h>
+#endif
+// Reset the device if watchdog_handle is not called within roughly this time
+// period.
+//
+// Options are: 15ms, 30ms, 60ms, 120ms, 250ms, 500ms, 1s, 2s, 4s, 8s.
+//
+// TODO: This value was not chosen carefully.
+#define WATCHDOG_TIMEOUT WDTO_120MS
```

Comment:
I don't love having this be a #define in the header, because now the macro `WATCHDOG_TIMEOUT` is in every file which includes `hal.h`.

Perhaps just hardcode WDTO_120MS in `watchdog_init()` (along with a comment)?  Or if you want it to be a constant, something like the following should work:

```
class HalApi {
  private:
    static constexpr inline auto WATCHDOG_TIMEOUT = WDTO_120MS;
}
```
I suggest auto because we don't really know what WDTO_120MS is -- is it a number, a pointer, ...?  But since I happen to know it's a number, you could also make it an `int`, that's fine too.